src/mote/templates/mainpage.html

```diff
@@ -394,5 +395,43 @@ <h6 class="modal-title head fw-bold">Recent conversations</h6>
                 </div>
             </div>
         </div>
+        <div class="toast-container"> 
+            <div id="parent" class="position-fixed bottom-0 end-0 p-3" style="z-index: 11">
+            </div>
+        </div>
     </body>
+    <script>
+      var socket = io.connect()
+      function showToast(meeting_json){
+            meeting_count = meeting_json.length
+            const timer = ms => new Promise(res => setTimeout(res, ms))
+            async function load() {
+            for (var i = 0; i < meeting_count; i++) {
+                var toastDiv = $(`
+                    <div class="toast-body">
+                        <h6 id="meeting-name">`+ meeting_json[i]["msg"]["meeting_topic"] +`</h6>
+                        <div class="mt-2 pt-2 border-top">
+                            <a id="toast-btn" type="button" class="btn btn-primary btn-sm" href="`+ meeting_json[i]["msg"]["url"] +`" target="_blank">Open meeting</a>
+                            <button type="button" class="btn btn-secondary btn-sm" data-bs-dismiss="toast">Close</button>
+                        </div>
+                    </div>
+                `)
+                var parentDiv=document.getElementById('parent')
+                var div = document.createElement('div')
+                div.setAttribute('class','toast')
+                div.setAttribute('role','alert')
+                div.setAttribute('aria-live','assertive')
+                div.setAttribute('aria-atomic','true')
+                $(div).append(toastDiv)
+                $(parentDiv).append(div)
+                var toast = new bootstrap.Toast(div)
```

Comment:
One way of increasing the delay (in ms):
```suggestion
                var toast = new bootstrap.Toast(div, {delay: 10000})
```